index.html

```diff
@@ -1,312 +0,0 @@
-                                                continuous
-                                                systems,
-                                                including random
-                                                vibrations, nonlinear systems, and aeroelastic phenomena.
-                                          </p>
-                                          <div class="linkButton">
-                                                <button id="pic-link-button"><a href="blogPost.html"
-                                                            target="_blank">Learn more!</a></button>
-                                          </div>
-                                    </div>
-
-                              </div>
-                        </div>
-                  </div>
-            </div>
-
-            <div class="footerContainer">
-                  <div class="footer">
-
-
-                        <div class="footerTitle">QUESTIONS OR COMMENTS? <SPAN id="footerTitleBold">GET IN TOUCH:</SPAN>
-
-                        </div>
-
-                        <div class="formSection">
-
-                              <form action="action_page.php">
-                                    <div>
-                                          <input type="text" id="fname" name="firstname" placeholder="Your name.."
-                                                required>
-
-                                          <input type="text" id="email" name="lastname" placeholder="Your email"
-                                                required>
-                                    </div>
-                                    <label class="subject" for="subject"></label>
-                                    <textarea id="subject" name="subject" placeholder="Write something.."
-                                          style="height:200px"></textarea>
-                                    <input type="submit" class="submitButton" value="Submit">
-
-
-                        </div>
-                        <div class="footerItem footerText">We're a most promising species, Mr. Spock, as predators
-                              go.
-                              Did you know that?
-                              I
-                              frequently have my doubts. I don't. Not any more. And maybe in a thousand years or
-                              so,
-                              we'll be able
-                              to prove it. To all mankind -- may we never find space so vast, planets so cold,
-                              heart and
-                              mind so
-                              empty that we cannot fill them with love and warmth. It's time you learned that
-                              freedom is
-                              never a
-                              gift. It has to be earned.
-                        </div>
-
-
-                        <div class="footerMediaLink footerAddress">

```

Comment:
nice use of classes to style things that should have a similar, or the same look